assets/js/src/form_editor/templates/selection.tsx

```diff
@@ -1,47 +1,72 @@
-import React from 'react';
-import Heading from 'common/typography/heading/heading';
+import React, { useState } from 'react';
 import MailPoet from 'mailpoet';
 import { useSelect, useDispatch } from '@wordpress/data';
 import { Button } from '@wordpress/components';
-
-import { TemplateType } from './store/types';
+import Categories from 'common/categories/categories';
+import Background from 'common/background/background';
+import { TemplateData } from './store/types';
 
 export default () => {
-  const templates: Array<TemplateType> = useSelect(
+  const [selectedCategory, setSelectedCategory] = useState('popup');
```

Comment:
Why do you keep the selected category in the local state rather than passing it to the store?